### DISCUSSION TITLE
HyperTrade - Distributed cryptocurrency trading bot

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#44] [Comments]
# HyperTrade - Distributed cryptocurrency trading bot

HyperTrade is a ready to deploy, distributed cryptocurrency trading bot

https://www.karanpratapsingh.com/blog/hypertrade-distributed-cryptocurrency-trading-bot

**Comment by riskiip** [comment]
Wow cool. Nice one dude
